docs/getting-started/supported-languages-and-tools.md

```diff
@@ -262,6 +263,13 @@ The table below lists all programming languages that Codacy supports and the cor
       <td><a href="https://pmd.github.io/pmd/pmd_userdocs_cpd.html">PMD CPD</a></td>
       <td><a href="https://github.com/realm/SwiftLint">SwiftLint</a><a href="#swiftlint-complexity"><sup>2</sup></a></td>
     </tr>
+    <tr>
+      <td>Terraform</td>
+      <td><a href="https://github.com/codacy/codacy-checkov">Checkov</a></td>
```

Comment:
@pauloribeiro-codacy does this repo uses some kind of Dictionary where I should add Checkov, or should I ignore it Codacy side?

Reply:
@lolgab found it! thx! :D

Reply:
Beautiful. :+1: